### DISCUSSION TITLE
Godot 3.5

### DISCUSSION THREAD

**Discussion by IanBUK** [dotnet-bluetooth-le/dotnet-bluetooth-le#696] [Q&A]
Does this library work with Godot 3.5?

I want to have a cross platform library available for use within my Godot app, preferably called from the c# code I’m using for my scripting.

thanks…

Ian.

**Comment by janusw** [comment]
Actually I don't know the Godot framework, but if it supports .NET Standard 2.0 or .NET 6/7, then it should probably work with this library, I assume. In case you try, maybe you can share your results with us? Which hardware/OS platforms would you be targeting?

**Comment by IanBUK** [comment]
I’ll certainly share results.I’m hoping to get my app running on Android tablets and iPads, with the appropriate builds and platform specific code. I’m doing Android first to put off paying the Apple Developer fee - £80 a year - for a while.I’ll have a play this weekend.On 5 Jul 2023, at 10:36, Janus Weil ***@***.***> wrote:﻿
Actually I don't know the Godot framework, but if it supports .NET Standard 2.0 or .NET 6/7, then it should probably work with this library, I assume. In case you try, maybe you can share your results with us? Which hardware/OS platforms would you be targeting?

**Comment by IanBUK** [comment]
The Godot website says that version 4 is compatible with .net7, so I’ll have a play over the weekend.IanOn 5 Jul 2023, at 10:36, Janus Weil ***@***.***> wrote:﻿
Actually I don't know the Godot framework, but if it supports .NET Standard 2.0 or .NET 6/7, then it should probably work with this library, I assume. In case you try, maybe you can share your results with us? Which hardware/OS platforms would you be targeting?
